app/constants/application-urls.js

```diff
@@ -0,0 +1,7 @@
+const { applicationApiUri } = require('../config')
+
+module.exports.APPLICATION = `${applicationApiUri}/application`
+module.exports.APPLICATION_CLAIM = `${applicationApiUri}/application/claim`
+module.exports.APPLICATION_GET = `${applicationApiUri}/application/get`
+module.exports.APPLICATION_SEARCH = `${applicationApiUri}/application/search`
+module.exports.CLAIM_BY_REFERENCE = `${applicationApiUri}/claim/get-by-reference`
```

Comment:
Is it worth adding all the URLs defined in the api files above into this file so they're in one place?

Reply:
I added this file at the beginning when I was taking a slightly different approach an moving the calls to the application api up into their handlers. Realised that this would have a significant impact on our testing so (for now at least) I've left them in their api folders and passed the logger down to them. So think it's reasonable to just leave the single use strings in the api call files.

TL;DR: I intend to remove this file.